content/support.rst

```diff
@@ -0,0 +1,212 @@
+
+	 * SFTPPlus product version?
+
+	 * If Server – do you also use WebAdmin?
+
+	 * Operating system name?
+
+	 * Operating system version?
+
+	 * CPU architecture?
+
+	 * File transfer protocol(s) used?
+
+	 * Is this installation part of a resilience/redundancy/high availability architecture?
+
+	 * Other information (if known). Please advise brief details of any
+	 integration, software, security or other technologies that interact
+	 with SFTPPlus products. e.g. Firewalls, Proxy, LDAP, Active Directory,
+	 Single Sign On, Management software etc
+
+	Before releasing a new version of SFTPPlus products or any fix, we run
+	an automated test suite to validate new features and check for
+	regressions. By knowing a few details about your installations we can
+	configure the test suite to run against a setup similar to your
+	installation. This information will not be shared with any other third
+	party and will be used solely for improving further support for you and
+	development of the product.
+
+
+Using the installation details
+------------------------------
+
+Once the installation details are provided, recreate the issue in a similar
+environment.
+
+Access the buildslave page of the OS to obtain the IP address and SSH into
+the IP.
+
+Download the initialize the SFTPPlus version that is the same as the customer's
+and initialise.  Do not test with the buildslave version.
+
+
+Knowledge Base
+==============
+
+Currently the main knowledge base is in documentation - both on the website
+and in the internal documentation pages supplied with each package.
+
+A separate knowledge base is currently under evaluation.
+
+
+Chat
+====
+
+`Olark <https://www.olark.com>`_ is currently used for chat.
+
+Each chat transcript is forwarded to the Support email list and subsequently
+creates a new ticket in the helpdesk.
+
+
+Licenses and support contract
+=============================
+
+See the `Support options page <https://www.sftpplus.com/support/options.html>`_
+which describes what options are available.
+If a support request goes beyond the standard option (such as a new
+product feature), the Sales team will determine the quote.
+
+See the `Life cycle page <https://www.sftpplus.com/product/life-cycle.html>`_ 
+for details about the product life cycle.
+
+If a feature request is obtained, check that it is included in the
+`Product roadmap <https://www.sftpplus.com/product/roadmap.html>`_ 
+
+License information is available from Sales team to check if a customer
+has an existing Support license.
+
+
+Support Request Management
+==========================
+
+
+Logging a support request
+-------------------------
+
+Support requests can be received either through the online support form,
+direct email or phone call.
+
+Email and support forms are the preferred methods.
+This is due to the possibility of miscommunication that can happen in a
+phone call.
+
+Support requests can be logged by partners, customers or the Support team.
```

Comment:
Both are the same.

Recieved: You have recieved it + It creates a ticket on Freshdesk
Logged - You have recieved it + It creates a ticket on Freshdesk + Includes additional actions like creating a Trac ticket if a defect is found.